react-native/codepush-hockeyapp/appcenter-post-build.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Example: Install App Center CLI
+npm install -g appcenter-cli
+
+ENVIRONMENT=Development
+APP_NAME=OrganizationName/MyApp-iOS
+APP_PATH=myApp.ipa
+# HockeyApp-iOS-Dev
+HOCKEY_APP_ID=""
+HOCKEY_TOKEN=""
+APP_CENTER_TOKEN=""
+COMMIT_MESSAGE=$(git log -1 HEAD --pretty=format:%s)
+
+if [ "$APPCENTER_BRANCH" == "master" ]; then
```

Comment:
The selection of which environment to use might be more streamlined, i.e. defining app names for iOS & Android in the section above, then just using those values.
It would be a nice improvement, though I don't think really required, since that is mostly a template to be customized by users of the script.

Reply:
@ranterle perfect, very clear! check the last changes my friend! 👍